google-ads-examples/src/main/java/com/google/ads/googleads/examples/targeting/GetGeoTargetConstantsByNames.java

```diff
@@ -60,41 +60,49 @@ public static void main(String[] args) {
     }
   }
 
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
   private void runExample(GoogleAdsClient googleAdsClient) {
-    GeoTargetConstantServiceClient geoTargetClient =
-        googleAdsClient.getLatestVersion().createGeoTargetConstantServiceClient();
+    try (GeoTargetConstantServiceClient geoTargetClient =
```

Comment:
This is the key change for this example. Ensures the service client is properly closed upon completion.